### DISCUSSION TITLE
Masonry Grid

### DISCUSSION THREAD

**Discussion by atumas-bananamilk** [mantinedev/mantine#2792] [Feature requests]
Could we get something like this in Mantine.
It's called Masonry Grid.

https://mui.com/material-ui/react-masonry

![Screenshot 2022-10-25 at 07 22 44](https://user-images.githubusercontent.com/22228431/197682023-6b725993-8d20-4a55-9779-ca9a666034ea.png)

**Comment by joshxyzhimself** [comment]
is this possible? really nice!
